copr_script.sh

```diff
@@ -35,9 +34,7 @@ readonly NAME="hardened-chromium-subresource-filter"
 cp $NAME/$NAME.spec ./
 cp $NAME/install_filter.sh ./
 cp $NAME/chromium-latest.py ./
-python3 ./chromium-latest.py --version $VERSION --stable --cleansources
-rm chromium-$VERSION.tar.xz
-rm -rf ./chromium-$VERSION
+cp /usr/src/chromium/chromium-*-clean.tar.xz ./
```

Comment:
this will exist already i assume because we tell copr config to install it from the specified repo?